dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -116,22 +116,12 @@ public List<Enrollment> getEnrollments(@Nonnull Set<UID> uids) throws ForbiddenE
       return List.of();
     }
 
-    EnrollmentQueryParams queryParams;
     try {
-      queryParams =
-          paramsMapper.map(
-              EnrollmentOperationParams.builder().enrollments(uids).build(),
-              getCurrentUserDetails());
+      return getEnrollments(EnrollmentOperationParams.builder().enrollments(uids).build());
     } catch (BadRequestException e) {
       throw new IllegalArgumentException(
           "this must be a bug in how the EnrollmentOperationParams are built");
     }
-
-    return getEnrollments(
-        new ArrayList<>(enrollmentStore.getEnrollments(queryParams)),
-        EnrollmentParams.FALSE,
-        false,
-        queryParams.getOrganisationUnitMode());
```

Comment:
why do we not need the org unit mode anymore?

Reply:
We still need it.
This change do not remove it, it just creates the `EnrollmentOperationParams` and call the `getEnrollments` method instead of the manually creating `EnrollmentQueryParams` and calling the store.
If you go inside `getEnrollments` method you will see that this code is still there.

Reply:
I just meant it in this particular case, but I guess it's why there's no org unit param from the user so we end up using the default one, right?

Reply:
The org unit mode is needed to build the `EnrollmentQueryParams` and if it wasn't previously set, then the default one is used.
I just wanted to be clear because of your `anymore` on the first question, nothing changed with this refactor.